### PR TITLE
Cassandra missing store features

### LINKED ISSUE #171: Add missing cassandra store features
Missing:
- [ ] `addColumn`
- [ ] `dropColumn`
- [ ] `updateColumnType`

### PR DESCRIPTION
**Pull request by schoenja**
Fixes #171

### CODE REVIEW

**Comment by schoenja** [comment]
> Can you please add a implementation for `updateColumnType`.

Sadly not really doable right now, as Cassandra doesn't support it. It actually makes it very difficult to work around this limitation, due to the fact that dropping and recreating the column with the same name but different type is also not supported.

The best way to solve this would be using distinct column placements, which in turn have a physical name assigned to them. I have opened an issue to discuss the possibility of this idea: #177 

If column placement with independent physical names was possible, one could append a revision number to the physical column, e.g. `col56_r2` turns into `col56_r3`. This avoids the name conflict.

**Comment by schoenja** [comment]
Blocked till #182 is merged.

**Comment by vogti** [comment]
#182 is merged.

**Comment by schoenja** [comment]
@vogti updateColumnType has been implemented. The implementation is not perfect, but it'll work for now.

File: cassandra-adapter/src/main/java/org/polypheny/db/adapter/cassandra/CassandraPhysicalNameProvider.java [hunk 1]
```diff
@@ -18,27 +18,185 @@
...
+    public String generatePhysicalSchemaName( int schemaId ) {
+        // TODO JS: implement cassandra schemas
+        return "cassandra";
+//        return "sch" + schemaId;
+    }
+
+
+    public String getPhysicalSchemaName( int schemaId ) {
+        return generatePhysicalSchemaName( schemaId );
+    }
```
Comment (vogti) [inline cassandra-adapter/src/main/java/org/polypheny/db/adapter/cassandra/CassandraPhysicalNameProvider.java:66]: Why not returning `DEFAULT_SCHEMA` here?

File: cassandra-adapter/src/main/java/org/polypheny/db/adapter/cassandra/CassandraPhysicalNameProvider.java [hunk 2]
```diff
@@ -18,27 +18,185 @@
...
+
+
+    public String getPhysicalColumnName( long columnId ) throws RuntimeException {
+        // TODO JS: This really should be a direct call to the catalog!
+        List<CatalogColumnPlacement> placements;
+        try {
+            placements = catalog.getColumnPlacementsOnStore( this.storeId );
+        } catch ( GenericCatalogException e ) {
+            throw new RuntimeException( e );
+//            e.printStackTrace();
```
Comment (vogti) [inline cassandra-adapter/src/main/java/org/polypheny/db/adapter/cassandra/CassandraPhysicalNameProvider.java:81]:
```suggestion
```
Remove those

File: cassandra-adapter/src/main/java/org/polypheny/db/adapter/cassandra/CassandraPhysicalNameProvider.java (hunk 1 above)
Comment (schoenja) [reply cassandra-adapter/src/main/java/org/polypheny/db/adapter/cassandra/CassandraPhysicalNameProvider.java:67]: `DEFAULT_SCHEMA` refers to the default logical schema name, not the physical schema name. The constant should probably reflect that.

**Pull request merged**